### PR TITLE
updated client library api to 0.16.4

### PR DESCRIPTION
**Pull request by ptone**
removed beta flag from gcloud pubsub api

### CODE REVIEW

**Comment by parkjam4** [comment]
No changes are needed from a writing perspective. But will need someone to review the code changes.

**Comment by ptone** [comment]
@jmdobry can you assign a code reviewer?

File: tutorials/cloud-functions-sns-pubsub/index.js [hunk 1]
```diff
@@ -46,7 +46,6 @@ exports.receiveNotification = function receiveNotification (req, res) {
     if (err) {
       // the message did not validate
       res.status(403).end('invalid SNS message');
-      return;
```
Comment (jmdobry) [inline tutorials/cloud-functions-sns-pubsub/index.js]: I don't think it's a good idea for the `return` statements to be removed, but otherwise LGTM

Comment (ptone) [reply tutorials/cloud-functions-sns-pubsub/index.js]: This seems to be generally the pattern in https://github.com/firebase/functions-samples

Comment (jmdobry) [reply tutorials/cloud-functions-sns-pubsub/index.js]: I'm sorry but it's not. I just looked through that repo, and they do one of two things:

1) Wrap all branch conditions in either `if`, `if else`, or `else`, ensuring that they are mutually exclusive.
2) Wrap all branch conditions except that last one in an `if` statement only, including a `return` statement at the end of each `if` to ensure other branches aren't inadvertently executed.

Both methods ensure that only one code path of the function is executed. The code in this PR originally used method #2, but now is opening itself up to the possibility of multiple branches being executed within a single function execution. You either need to leave the `return` statements in place, or switch to method #1 combining `if`, `if else`, and `else` to ensure the code paths remain mutually exclusive.

**Comment by ptone** [comment]
@jmdobry OK - returns returned - I was a little fuzzy on when why they were in some paths but not others but think I get it now (had seenhttps://github.com/GoogleCloudPlatform/community/pull/214/files remove them from this tutorial)

**Pull request merged**